src/Algolia.Search/Algolia.Search.csproj

```diff
@@ -23,8 +23,15 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <TargetFrameworks>netstandard2.1;netstandard2.0;netstandard1.3</TargetFrameworks>
     <LangVersion>7.3</LangVersion>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1">
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
```

Comment:
Hey @sbellone, I suppose you are talking about the IncludeAssets element. It was added by the CLI when I added the package. You can safely replace it with `PrivateAssets="All"` to exactly match the doc. It should be equivalent.

If your question was about the relevance of the Microsoft.SourceLink.GitHub package itself, then yes it is important for the IDE to download the source code from github while debugging as it will embed the source files url in your package.